jenkinsfiles/dev

```diff
@@ -182,6 +182,41 @@ pipeline {
                 }
             }
         }
+
+        stage('Update IM Play dev instance') {
+            if (env.GIT_BRANCH == 'master') {
+                steps {
+                    script {
+                        withCredentials([usernamePassword(credentialsId: 'e2e-im-user', passwordVariable: 'PASSWORD', usernameVariable: 'USER_EMAIL')]) {
+                            dir('im-db-manager') {
+                                sparseCheckout('https://github.com/dhis2-sre/im-database-manager', 'master', '/scripts')
+
+                                dir('/scripts') {
+                                    env.DATABASE_ID = sh(
+                                            returnStdout: true,
+                                            // TODO: How do we avoid hard coding the name/version?
+                                            script: "./list.sh | jq -r '.[] | select(.Name == \"whoami\") | .Databases[] | select(.Name == \"Sierra Leone - 2.39.0.sql.gz\") | .ID'"
+                                    ).trim()
+
+                                    sh '[ -n "$DATABASE_ID" ]'
+                                    echo "DATABASE_ID is ${env.DATABASE_ID}"
+                                }
+                            }
+
+                            dir('im-manager') {
+                                sparseCheckout('https://github.com/dhis2-sre/im-manager', 'master', '/scripts')
+
+                                dir('im-manager/scripts') {
```

Comment:
It should be just `scripts` here without the leading `im-manager/`, as we're already in the `im-manager` dir at that point.

Reply:
Same as above